src/living_playbook.json

```diff
@@ -3604,4 +3607,4 @@
       "uid": 284
     }
   ]
-}
\ No newline at end of file
+}
```

Comment:
sorry, GitHub automatically added this newline when I edited the file in the web editor. I'm happy to clean it up but figured it would come back later when other people edit it this way...

Reply:
ok, good! I'll go in and change all the spaces to tabs now